toolchain/check/testdata/var/no_prelude/decl_with_init.carbon

```diff
@@ -30,12 +30,20 @@ fn Main() {
 // CHECK:STDOUT:
 // CHECK:STDOUT: fn @Main() {
 // CHECK:STDOUT: !entry:
+// CHECK:STDOUT:   name_binding_decl {
+// CHECK:STDOUT:     %x.patt: %empty_tuple.type = binding_pattern x
+// CHECK:STDOUT:     %.loc12_3.1: %empty_tuple.type = var_pattern %x.patt
+// CHECK:STDOUT:   }
 // CHECK:STDOUT:   %x.var: ref %empty_tuple.type = var x
```

Comment:
The order of the output IR here is a bit surprising: first we create the var, then we initialize it (perhaps in-place), and only then do we evaluate the type expression that's involved in computing the type of the var. We're getting away with this because we can directly reference the type constant without mentioning the computation that produced it, but we should probably reorder these eventually so the type computation is spliced in prior to the `var` storage instruction.